boot/freeldr/freeldr/settings.c

```diff
@@ -1,9 +1,10 @@
 /*
- * PROJECT:         ReactOS Boot Loader
- * LICENSE:         BSD - See COPYING.ARM in the top level directory
- * FILE:            boot/freeldr/freeldr/cmdline.c
- * PURPOSE:         FreeLDR Command Line Parsing
- * PROGRAMMERS:     ReactOS Portable Systems Group
+ * PROJECT:     FreeLoader
+ * LICENSE:     BSD - See COPYING.ARM in the top level directory
+ *           or MIT (https://spdx.org/licenses/MIT)
```

Comment:
It can be both? Looks complicated. 🤔

Reply:
I have no idea how to write the header to tell it's "dual licensed" in the usual sense.

Reply:
Oh, derp. 👍